ocean_lib/ocean/ocean_exchange.py

```diff
@@ -104,7 +104,7 @@ def create(self, data_token: str, exchange_rate: float, wallet: Wallet) -> str:
 
         return exchange_id
 
-    def setRate(
+    def set_rate(
         self,
```

Comment:
The instructions are to preserve labels of methods to be 1:1 with Solidity code.

This is a 1:1 for setRate() in fixedRateExchange.sol https://github.com/oceanprotocol/contracts/blob/main/contracts/fixedRate/FixedRateExchange.sol

Reply:
Sorry, will modify right now as how it was.